dll/cpl/desk/classinst.c

```diff
@@ -4,7 +4,7 @@
  * FILE:            dll/cpl/desk/classinst.c
  * PURPOSE:         Class installers
  *
- * PROGRAMMERS:     Herv� Poussineau (hpoussin@reactos.org)
+ * PROGRAMMERS:     Hervé Poussineau (hpoussin@reactos.org)
```

Comment:
Check your file's codepage, this shouldn't have happened.

Reply:
This is a conversion _to_ UTF8, so is fine.